doc/pages/tutorials/010-astarte_in_5_minutes.md

```diff
@@ -43,7 +43,7 @@ Docker version 19.03.8
 Docker compose version >= 1.17 is recommended:
```

Comment:
Here it should be added that Docker compose V2 (the one without the `-`) is required.
```suggestion
Docker compose version >= 2.21 is recommended:
```